modules/analytics/src/views/full/index.tsx

```diff
@@ -188,6 +199,23 @@ const Analytics: FC<any> = ({ bp }) => {
     })
   }, [state.dateRange, state.selectedChannel])
 
+  useEffect(() => {
+    const metrics = orderMetrics(getMetric('msg_sent_qna_count'))
+      .filter(r => r.name)
+      .slice(0, 10)
+
+    // tslint:disable-next-line: no-floating-promises
+    Promise.all(metrics.map(r => fetchQnaQuestion(r.name.replace('__qna__', '')))).then(values => {
+      const qnaQuestions = values.reduce((acc, { data: { questions }, id }) => {
+        const question = questions[lang.getLocale()][0]
+        acc[`__qna__${id}`] = question
+        return acc
+      }, {})
+
+      dispatch({ type: 'receivedQnaQuestions', data: { qnaQuestions } })
+    })
```

Comment:
Can you move this outside of the useEffect and use async/await ? It would make this block more clear

Reply:
Done in 13eb5c6